src/services/local-cache/index.js

```diff
@@ -0,0 +1,39 @@
+import { globalConfig } from '../../services/config.js'
```

Comment:
Suggestion:
On the frontend we'd like to minimize the amount of code where we can since its pulled to the client to run.  I think having a single LocalCache class with all the isMA logic in it is good enough.

  async getItem(key) {
  if(isMA) return await storage.getItem(key);
  return storage.getItem(key);
  }